students/slupek_bartosz/lesson_03_functions/collatz.py

```diff
@@ -0,0 +1,15 @@
+def collatz(number):
+    if number % 2 == 0:
+        print(number // 2)
+        return number // 2
+    elif number % 2 == 1:
+        print(3 * number + 1)
+        return 3 * number + 1
+
+
+try:
+    a = int(input('Enter number: '))
+    while a != 1:
```

Comment:
lepszym warunkiem by było `a > 1`, bo jak dostaniemy 0 lub coś mniejszego, to chyba pętla w nieskończoność pójdzie